src/connexion/extended/auth_paths_extd.py

```diff
@@ -1,11 +1,20 @@
 import logging
 
+from aiohttp import web
+import aiohttp_jinja2
+import jinja2
+from aiohttp.web_exceptions import HTTPPermanentRedirect
+from aiohttp.web_middlewares import normalize_path_middleware
```

Comment:
If the aiohttp dependencies are extra, it will get an error when trying to import these packages.

Reply:
Is it a good idea to import inside the class?

Reply:
Maybe it is better to split it into files anyway.